host/cli/collect-debug-info.go

```diff
@@ -182,3 +189,29 @@ func captureSchedulerState(gist *Gist) error {
 	gist.AddFile("scheduler-state.json", string(body))
 	return nil
 }
+
+func captureSireniaMetadata(gist *Gist) error {
+	appliances := []string{"postgres", "mariadb", "mongodb"}
+	for _, appliance := range appliances {
+		meta, err := discoverd.NewService(appliance).GetMeta()
+		if err != nil {
+			continue // skip if no service, appliance may not be enabled
+		}
+		dstate := &state.DiscoverdState{
+			Index: meta.Index,
+			State: &state.State{},
+		}
+		if len(meta.Data) > 0 {
+			err := json.Unmarshal(meta.Data, dstate.State)
+			if err != nil {
+				return err
+			}
+			body, err := json.MarshalIndent(dstate, "", "    ")
+			if err != nil {
+				return err
```

Comment:
We should just continue on these errors so we try and get the other states (maybe log the error).